ios-app/UI/CustomTestGenerationViewController.swift

```diff
@@ -94,6 +126,14 @@ class CustomTestGenerationViewController: WebViewController, WKScriptMessageHand
         present(viewController, animated: true, completion: nil)
     }
     
+    func gotoTestReport(_ attempt: Attempt) {
+        let storyboard = UIStoryboard(name: Constants.EXAM_REVIEW_STORYBOARD, bundle: nil)
+        let viewController = storyboard.instantiateViewController(withIdentifier:
+                Constants.TEST_REPORT_VIEW_CONTROLLER) as! TestReportViewController
```

Comment:
This ViewController name itself is confusing(not sure if it means exam or test case). Would prefer it to be renamed in separate commit.